lib/ansible/playbook/base.py

```diff
@@ -259,7 +259,11 @@ def get_loader(self):
     def get_variable_manager(self):
         return self._variable_manager
 
-    def _validate_debugger(self, attr, name, value):
+    def _post_validate_debugger(self, attr, value, templar):
+        if templar.is_template(value):
+            value = templar.template(value, fail_on_undefined=True)
+        else:
+            value = value
```

Comment:
Remove this `if/else`, and instead just do `value = templar.template(value)`.  Also, we shouldn't force `fail_on_undefined`.  The caller will handle templating failures.

Reply:
I've submitted a change according to what you've suggested, thanks